assets/src/platform-admin/pages/RestrictedPage.tsx

```diff
@@ -1,14 +1,17 @@
 import * as React from "react";
-import { ErrorAlert } from "@pay/web";
-
+import { ErrorAlert, Loader } from "@pay/web";
 import { usePayUser } from "../../users";
 
 interface Props {
   children: JSX.Element;
 }
 
 const RestrictedPage: React.FC<Props> = ({ children }) => {
-  const user = usePayUser();
+  const { isLoading, user } = usePayUser();
+
+  if (isLoading) {
+    return <Loader message="Loading" />;
```

Comment:
@acodegirl minor, but the `message` prop is already `Loading` so no need to use the prop here.